content/millennium/dstu2/scheduling/appointment.md

```diff
@@ -47,26 +47,26 @@ Search for Appointments that meet supplied query parameters:
 
```

Comment:
Add a period at the end of line 12, and remove the br tags from line 13

Reply:
Addressed with f54f352a457572e639496cf1f96efa62816cd041